code/modules/mob/living/carbon/human/life.dm

```diff
@@ -462,7 +453,7 @@
 			stat = CONSCIOUS
 
 		//Eyes
-		if(disabilities & BLIND)	//disabled-blind, doesn't get better on its own
+		if(disabilities & BLIND || stat)	//disabled-blind, doesn't get better on its own

```

Comment:
Doesn't this mean that living mobs will never heal from temporary blindness? Why'd you do this?